superset-frontend/cypress-base/cypress/integration/dashboard/filter.test.ts

```diff
@@ -101,16 +101,19 @@ describe('Dashboard filter', () => {
     cy.get('.filter_box button').click({ force: true });
 
     // wait again after applied filters
-    cy.wait(aliases.filter(x => x !== getAlias(filterId))).then(requests => {
+    cy.wait(
+      aliases.filter(x => x !== getAlias(filterId)),
+      { timeout: 30000 },
```

Comment:
This seems like a reeeeeally long timeout. Is there a reason for this? If so, maybe add a comment to the code so people don't follow the pattern here unnecessarily, and potentially slow down a hundred failing tests by 30 seconds each ;)

Reply:
I found the problem is in typing propably, deleted timeout and set delay to "type" operation - delay between each key entered. Triggered run to verify this on CI.

Reply:
it still seems to be flaky... I will try to fix first failing assertion from latest pipeline